### PR TITLE
Adding walk map, build map, and ground height map

### PR DESCRIPTION
**Pull request by ebetica**
Also adding replayer tests. I ran `th dump_replay.lua -t $VM_IP` with a replay set up to be loaded, and confirmed that it was correct. Previously, there was a bug where the map was written out as raw bytes, and space delimited, so somehow the stream wasn't reading it in correctly.

### CODE REVIEW

File: examples/dump_replay.lua [hunk 1]
```diff
@@ -0,0 +1,100 @@
+require 'torch'
```
Comment (syhw) [inline examples/dump_replay.lua]: Can you put a header/comment that explains how to test / how to use this file?

Comment (syhw) [inline examples/dump_replay.lua]: Actually, we should maybe include a replay in the repo 😮 This way your `th dump_replay.lua -t $VM_IP` can be complemented with a specific replay setup for the sake of the example.

Comment (ebetica) [inline examples/dump_replay.lua]: I did include a replay in maps/ :)

Comment (syhw) [inline examples/dump_replay.lua]: Then can you include new 👓 for me? :D

Comment (edran) [inline examples/dump_replay.lua]: :rofl:

**Pull request merged**